docs/user-guide/configuration.md

```diff
@@ -29,6 +25,21 @@ $ tree
 $ terraform-docs .
 ```
 
+As of `v0.13.0`, The default name of the configuration file is `.terraform-docs.yml`.
```

Comment:
```
As of `v0.13.0`, The default name of the configuration ...
```

The new change here is misleading, it gives the impression that the default name of the config file has been added i `v0.13.0`, but the ordering was added in v0.13.0.